src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -68,6 +77,8 @@
 
     "music_control.metadata.description": "Music Control - Built-in Resource Pack",
 
+    "minecraft:biome.minecraft.old_growth_taiga": "Old Growth Taiga",
```

Comment:
Funny that this biome translation is missing in the minecraft lang file ^^

Reply:
Found out it was not missing, but that it's just a sound event used for both spruce and pine taiga. As I'm adding sound event per biome, it won't ever be used. I will add an event black list for this kind of case